File: rserratos6.java
// Raquel Serratos
// CSC 321
// 10/10/2024

import java.util.Scanner;

public class rserratos6{
	public static void main(String[] args){
		double x = 3.0;
		double y = 4.0;
		System.out.printf("\n%s: %f\n", "x+y*x/y-x (double version)", x+y*x/y-x);

		int w = 3;
		int v = 4;
		double result = w+v*w/v-w;
		System.out.printf("%s: %f\n", "x+y*x/y-x (int version)", result);

		System.out.printf("\n%s: %f\n", "-x-y/x*y+x (double version)", -x-y/x*y+x);

		result = -w-v/w*v+w;
		System.out.printf("%s: %f\n", "-x-y/x*y+x (int version)", result);

		System.out.printf("\n%s: %f\n", "x+y-x/y (double version)", x+y-x/y);
		
		result = w+v-w/v;
		System.out.printf("%s: %f\n", "x+y-x/y (int version)", result);
	}
}

File: rserratos6.c
// Raquel Serratos

#include<stdlib.h>
#include<stdio.h>
#include<math.h>

int main(void){
	double x = 3.0;
	double y = 4.0;
	double result = 0;

	result = x+y*x/y-x;

	printf("First result using double: %lf \n", result);

	int w = 3;
	int v = 4;

	result = w+v*w/v-w;

	printf("Second result using int: %lf \n", result);
	
	result = -x-y/x*y+x;

	printf("\nFirst result using double: %lf \n", result);

	result = -w-v/w*v+w;

	printf("Second result using int: %lf \n", result);

	result = x+y-x/y;

	printf("\nFirst result using double: %lf \n", result);

	result = w+v-w/v;

	printf("Second result using double: %lf \n", result);

	return EXIT_SUCCESS;
}
